Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cover_crypt"
-version = "4.1.1"
+version = "4.2.0"
```

Comment:
I wonder if we should update to `5.0.0` instead, as we are widely breaking public API. What do you think?

Reply:
Yes, you are right... I was a bit worried to see the version number grow so fast.